python/fire/cfg/_process.py

```diff
@@ -91,6 +91,76 @@ def __init__(self, pass_name):
         from . import _rnss
         _rnss.RandomNumberSeedService()
 
+    def __setattr__(self, name, value) :
+        """Override setting attributes to allow for legacy translation and wrapping
+        of helper classes
```

Comment:
Document changes here and point out what should be kept/deleted upon removal of legacy processors.